common/utils/localStorage.ts

```diff
@@ -40,16 +40,16 @@ export async function saveWalletConfig(
   wallet: IWallet,
   state: Partial<WalletConfig>
 ): Promise<WalletConfig> {
-  const oldState = await loadWalletConfig(wallet);
+  const oldState = loadWalletConfig(wallet);
   const newState = { ...oldState, ...state };
-  const key = await getWalletConfigKey(wallet);
+  const key = getWalletConfigKey(wallet);
```

Comment:
this function no longer needs to be async

Reply:
thanks, fixed.